src/tree/PostgresAccountWizard/PostgresServerCredStep.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Huh I expected these files to be in the "postgres" folder. Can we put them somewhere in there?

Reply:
Do you mean the whole PostgresAccountWizard folder?

Reply:
Yes